usaspending_api/common/tests/integration/test_decorators.py

```diff
@@ -17,14 +12,14 @@ def test_statement_timeout_successfully_times_out():
     Test the django statement timeout setting
     """
 
-    test_timeout_in_seconds = 1
+    test_timeout_in_seconds = 0.5
```

Comment:
Reduced the wait time for all three tests, will save hours of test runtime on Travis over the course of a year